content/community/sidebars.ts

```diff
@@ -0,0 +1,121 @@
+module.exports = {
+    learn: [
+      {
+        type: 'category',
+        label: 'Funding',
+        collapsed: true,
+        items: ['funding/edf-funding', 'funding/community-treasury'],
+      },
+      {
+        type: 'category',
+        label: 'The Community',
+        collapsed: true,
+        items: [
+          'the-community/how-to-support',
+          'the-community/discord',
+          'the-community/x-teams',
+          'the-community/git-hub',
+          'the-community/social-media',
+        ],
+      },
+      {
+        type: 'category',
+        label: 'Research',
+        collapsed: true,
+        items: [
+          'research/research-outline',
+          'research/research-papers',
+          {
+            type: 'link',
+            label: 'Coordicide Specs',
+            href: '/IOTA-2.0-Research-Specifications/Preface'
+          },
+          {
+            type: 'link',
+            label: 'GoShimmer',
+            href: '/goshimmer/welcome'
+          },
+          'research/iota-devnet-wallet',
+        ],
+      },
```

Comment:
This feels a bit out of place here, though it doesn't really fit in `develop` either

Reply:
UX suggested moving it to `Learn`. Although I would still prefer having it in the Navbar (😢), I can live with that 👍